core/src/main/java/org/apache/accumulo/core/clientImpl/ReplicationClient.java

```diff
@@ -56,7 +56,7 @@
       log.debug("Could not get ReplicationCoordinator connection to {}, will retry",
           context.getInstanceName());
       try {
-        Thread.sleep(attempts * 250);
+        Thread.sleep((long) attempts * 250);
```

Comment:
I just realized this is another false positive, they should be able to detect since the loop control variable is bounded.  I will report it.